### DISCUSSION TITLE
Publishing app on izzyondroid

### DISCUSSION THREAD

**Discussion by NikunjK8** [ReadYouApp/ReadYou#749] [General]
Do you have any plans to release the app on Izzy repo? The reason why I'm asking this is because f-droid builds are always heavily delayed. It's been a week since 0.10 release

**Comment by Ashinch** [comment]
Hi, @NikunjKhangwal 

> It's been a week since 0.10 release

Due to the fatal bug in 0.10.0, it was skipped and the FDroid will be updated directly to 0.10.1, which should take about three days.

> Do you have any plans to release the app on Izzy repo?

I would like to publish my app in any open source repository, but sometimes they have more stringent apply procedures and requirements, which takes extra time, and I prefer to have experienced people help me with these things, you can open an issue, and see if anyone can do the job. 😉

**Reply by NikunjK8** [reply]
> Due to the fatal bug in 0.10.0, it was skipped and the FDroid will be updated directly to 0.10.1, which should take about three days.

Makes sense, thanks for the info

> I would like to publish my app in any open source repository, but sometimes they have more stringent apply procedures and requirements, which takes extra time, and I prefer to have experienced people help me with these things, you can open an issue, and see if anyone can do the job. 😉

Maybe Izzy himself can help? He's very active here. I can tag him if that helps.
